sdk/resourcemanager/Azure.ResourceManager/api/Azure.ResourceManager.netstandard2.0.cs

```diff
@@ -578,11 +578,12 @@ public partial class GenericResourceCollection : Azure.ResourceManager.ArmCollec
         public virtual Azure.Response<Azure.ResourceManager.Resources.GenericResource> Get(Azure.Core.ResourceIdentifier resourceId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Resources.GenericResource>> GetAsync(Azure.Core.ResourceIdentifier resourceId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
-    public partial class GenericResourceData : Azure.ResourceManager.Resources.Models.TrackedResourceExtendedData
+    public partial class GenericResourceData : Azure.ResourceManager.Models.TrackedResourceData
```

Comment:
Because of this change, we will have to change our implementation to keep its previous base classes